### PR TITLE
added autotask.net to dector hints

### PR DESCRIPTION
**Pull request by SrS2225a**
Adds dark theme detector support for autotask.net PSA software (enabled by user settings)

### CODE REVIEW

**Comment by Myshor** [comment]
Hard to review as it needs access to enter.

Can you at least show some screens with browser developer tools with that CSS included in dark mode that can confirm your proposal?

**Comment by SrS2225a** [comment]
Of course. Here are a few:

image
image

With dark reader on using the same element:
image

Without autotask dark mode enabled, which becomes `LandingDatto.min.css`:
image

Let me know if you need anymore information. Also, since the version number changed (v get parameter) since my initial pr, I will push up the fix later tonight.

File: src/config/detector-hints.config [hunk 1]
```diff
@@ -90,7 +90,7 @@ TARGET
 html
 
 MATCH
-link[href*="/Mvc/FrameworkContent/Styles/LandingMidnight.min.css?v=71278"]
+link[href*="/Mvc/FrameworkContent/Styles/LandingMidnight.min.css?v=72248"]
```
Comment (Myshor) [inline src/config/detector-hints.config:93]: If version changes maybe change to:
`link[href*="/Mvc/FrameworkContent/Styles/LandingMidnight.min.css?"]`
`*=` means it looks for this part of text.

**Review by Myshor — CHANGES_REQUESTED** [review]

File: src/config/detector-hints.config (hunk 1 above)
Comment (SrS2225a) [reply src/config/detector-hints.config]: Thanks, should be good to go now

**Review by Myshor — APPROVED** [review]

**Pull request merged**